keanu-python/tests/test_vertex_operations.py

```diff
@@ -6,42 +6,58 @@
 
 ### Comparisons
 
-@pytest.mark.parametrize("lhs, rhs, expected_result", [
-    (Const(np.array([1., 2.])),    Const(np.array([1., -1.])), np.array([[True], [False]])),
-    (Const(np.array([1., 2.])),          np.array([1., -1.]) , np.array([[True], [False]])),
-    (      np.array([1., 2.]) ,    Const(np.array([1., -1.])), np.array([[True], [False]])),
-    (Const(np.array([1.    ])),                    1.        , np.array([[True]         ])),
-    (Const(np.array([    2.])),                    1.        , np.array([        [False]])),
-    (                    1.   ,    Const(np.array([1.     ])), np.array([[True]         ])),
-    (                    1.   ,    Const(np.array([    -1.])), np.array([        [False]])),
-    (Const(np.array([1., 2.])),    Const(np.array([1.     ])), np.array([[True], [False]])),
-    (Const(np.array([1.    ])),    Const(np.array([1.,  2.])), np.array([[True], [False]])),
-])
+
+@pytest.mark.parametrize(
+    "lhs, rhs, expected_result", [
+        (Const(np.array([1., 2.])), Const(np.array([1., -1.])),
+         np.array([[True], [False]])),
+        (Const(np.array([1., 2.])), np.array([1., -1.]),
+         np.array([[True], [False]])),
+        (np.array([1., 2.]), Const(np.array([1., -1.])),
+         np.array([[True], [False]])),
+        (Const(np.array([1.])), 1., np.array([[True]])),
+        (Const(np.array([2.])), 1., np.array([[False]])),
+        (1., Const(np.array([1.])), np.array([[True]])),
+        (1., Const(np.array([-1.])), np.array([[False]])),
+        (Const(np.array([1., 2.])), Const(np.array([1.])),
+         np.array([[True], [False]])),
+        (Const(np.array([1.])), Const(np.array([1., 2.])),
+         np.array([[True], [False]])),
+    ])
```

Comment:
is it possible to mark specific blocks as "don't touch this formatting"?

Reply:
Yes. With '# yapf: disable' and '# yapf: enable'. I will wrap this block.

Reply:
great!

please do all the similar bits in this file, with lists of parameters